assets/aml-benchmark/tests/data/process_sample_examples.jsonl

```diff
@@ -0,0 +1,29 @@
+{"name":"gsm8k", "question":"Janet\u2019s ducks lay 16 eggs per day. She eats three for breakfast every morning and bakes muffins for her friends every day with four. She sells the remainder at the farmers' market daily for $2 per fresh duck egg. How much in dollars does she make every day at the farmers' market?","answer":"Janet sells 16 - 3 - 4 = <<16-3-4=9>>9 duck eggs a day.\nShe makes 9 * 2 = $<<9*2=18>>18 every day at the farmer\u2019s market.\n#### 18"}
```

Comment:
Are the example inputs from some dataset? Do we have approval for hosting a subset of those dataset? cc: @EricWrightAtWork

Reply:
Yes, those that I have taken in examples, those are approved. But I have just added one row manually instead of downloading the data to test scripts. Would the approval check still needed in this case?

Reply:
I think it is fine then. I would prefer creating some mock examples by hand like 5 rows. But that is nitpicking if we have approval to use examples from the dataset. So, it should be fine then.